Sources/Extensions/UIView.swift

```diff
@@ -89,4 +89,70 @@ extension UIView {
                                             constant: value)
         addConstraint(constraint)
     }
+
+    func setUpEmptyListLabel(text: String) -> UILabel {
+        let emptyListLabel = UILabel(frame: CGRect(x: 0, y: 0, width: 268, height: 20.5))
+        emptyListLabel.text = text
+        emptyListLabel.numberOfLines = 0
+        emptyListLabel.lineBreakMode = .byWordWrapping
+        emptyListLabel.textAlignment = .center
+        emptyListLabel.accessibilityIdentifier = "EmptyListLabelAccessibilityIdentifier"
```

Comment:
What is common approach to accessibility identifiers naming?
I see you've changed `DoneButtonAccessibilityIdentifier` -> `doneButton` above, but still follow old approach here.
Naming should be solid across the project, I think